### DISCUSSION TITLE
加入mcmod模组下载源

### DISCUSSION THREAD

**Discussion by githubusera2034324** [Meloong-Git/PCL#3679] [讨论]
mcmod是国内模组作者上传模组的网站，加入mcmod源可以更方便地下载到国内模组

**Comment by hejiehao** [comment]
#1658

**Comment by wuliaodexiaoluo** [comment]
没有合适的 API 。。。

**Comment by zkitefly** [comment]
mcmod 那边暂未公开可用的 API，不过重生应该在做了（吧）

**Reply by mc-124** [reply]
API是api.mcmod.cn，但是没有公开调用方法

**Reply by Deep-Dark-Forest** [reply]
> 它似乎根本没有api（

Zkitefly透露，mcmod 那边正在制作 api 呢

**Comment by mc-124** [comment]
没有API，爬取的话下载那有人机验证爬不了，顶多爬个名字 描述 链接 图片之类的小玩意
